src/patterns/task-list-pages/index.md.njk

```diff
@@ -65,14 +65,52 @@ Where possible, allow users to complete tasks in any order. This will help them
 
 ## Research on this pattern
 
-Read a blog post about [testing and iterating the task list page pattern](https://designnotes.blog.gov.uk/2017/04/04/weve-published-the-task-list-pattern/).
+This pattern was originally developed and tested by a team at the Government Digital Service (GDS). 
 
-### <a name="next-steps"></a>Next steps
+The team built prototypes of task lists for 3 services, Register as a childminder, Learn to drive and Transport goods and tested them with 34 users over 5 rounds of research. 
+
+The pattern was iterated after each round of testing.
+
+You can read more about [testing and iterating the task list page pattern](https://designnotes.blog.gov.uk/2017/04/04/weve-published-the-task-list-pattern/).
+
+### Known issues and gaps
+
+There is evidence that the completed tag might be misleading to some users.
+
+In one instance, [a user saw a task marked ‘completed’ and thought this meant they had submitted their response](https://github.com/alphagov/govuk-design-system-backlog/issues/72#issuecomment-413159884). As a result, they didn’t continue to the ‘Check and submit your response’ task, and were later issued with a court order for not responding.
```

Comment:
This is an extreme case so probably fine to mention...but generally we wouldn't normally write anything about one user in one instance. It might be beneficial to frontload the severity of this outcome?

Reply:
Yeah this was a tricky one to balance. We were worried about emphasising the severity too much, because then there's a question about whether it should even be in the Design System.

Personally, I think having the court order mentioned at the end makes it sound serious enough, without front loading it - but would you consider this a blocking suggestion?

Reply:
Not blocking at all. I just wouldn't want any other component / pattern to have research documented on the Design System unless the same thing had been observed at least 5 times. This case is clearly an exception though.

Reply:
yeh its a good thing to call out, but in this case the mistake had very bad repercussions, and is backed up by similar issues being reported around people not fully understanding what needs to be 'completed'